src/components/headerPicture.tsx

```diff
@@ -1,11 +1,44 @@
 import React from "react";
-import "../app";
 import styled from "@emotion/styled";
+import Clipart from "../resources/clip-uploading.svg";
 
 const HeaderPictureContainer = styled.div`
   height: 19.5rem;
 `;
 
+const HeaderTextOne = styled.p`
+  font-style: normal;
+  font-weight: 600;
+  font-size: 64px;
+  line-height: 77px;
+  letter-spacing: -0.02em;
+  color: #2c3549;
+`;
+
+const HeaderTextTwo = styled.p`
+  font-style: normal;
+  font-weight: 400;
+  font-size: 28px;
+  line-height: 30px;
+`;
+
+const ImgContainer = styled.img`
+  padding-left: 14rem;
+  height: 310px;
+  object-fit: cover;
+`;
+
 export const HeaderPicture: React.FunctionComponent = () => {
-  return <HeaderPictureContainer className="test header-pic-main">test</HeaderPictureContainer>;
+  return (
+    <HeaderPictureContainer className="flex flex-row justify-between px-56">
+      <div className="mt-16">
+        <HeaderTextOne>Gallery</HeaderTextOne>
```

Comment:
why is everything in different part ? (different rendering depending on screen size below)

![image](https://user-images.githubusercontent.com/503185/73624768-0cedf880-467d-11ea-949a-9261ab8cfaf9.png)

![image](https://user-images.githubusercontent.com/503185/73624779-14ad9d00-467d-11ea-9354-516c99e43df6.png)